.gitignore

```diff
@@ -0,0 +1,2 @@
+venv
+diagrams
```

Comment:
I think i still want to leave diagrams/  directory for now so folks can see the output the script generates. We may decide to move it to the "examples" in the future.